op-node/rollup/derive/l1_block_info.go

```diff
@@ -30,14 +30,14 @@ type L1BlockInfo struct {
 	// Not strictly a piece of L1 information. Represents the number of L2 blocks since the start of the epoch,
 	// i.e. when the actual L1 info was first introduced.
 	SequenceNumber uint64
-	BatcherAddr    common.Address
-	L1FeeOverhead  [32]byte
-	L1FeeScalar    [32]byte
+	// BatcherHash version 0 is just the address with 0 padding to the left.
+	BatcherAddr   common.Address
```

Comment:
You mean `BatcherAddr`? Or is this referring to something else?

Reply:
The batcher hash is just `bytes32(uint256(uint160(address)))` aka left padding the 20 byte address with zeros to 32 bytes

Reply:
This PR explains it: https://github.com/ethereum-optimism/optimism/pull/3831

Its because this is versioned hash 0, where future versions may pack more data in via a commitment to the data